graylog2-server/src/test/java/org/graylog/plugins/pipelineprocessor/functions/FunctionsSnippetsTest.java

```diff
@@ -362,6 +362,15 @@ public static void registerFunctions() {
         functionRegistry = new FunctionRegistry(functions);
     }
 
+    @Test
+    public void string_concat(){
```

Comment:
Please change the name to `stringConcat` to look the same as the other tests.
That also requires the file `string_concat.txt` to be `stringConcat.txt` below.